avm/res/purview/account/main.json

```diff
@@ -5,8 +5,8 @@
   "metadata": {
     "_generator": {
       "name": "bicep",
-      "version": "0.25.53.49325",
```

Comment:
The readme change is good (be it curious) but I'd suggest to undo the JSON change as we'd not want to publish a new module version.